src/components/MySpace/MySpace.tsx

```diff
@@ -83,7 +83,7 @@ const MySpace = ({ bookmarks }: { bookmarks: BookmarkRecords }) => {
     )
 
     handleAddWidget({
-      variables: { title: 'Ideal Title', type: AddWidgetType.GuardianIdeal },
+      variables: { title: 'Guardian Ideal', type: AddWidgetType.GuardianIdeal },
```

Comment:
Updated this to match pattern for the other widgets. An objection to the title of this widget being "Guardian Ideal"?